apps/arweave/src/ar_merkle.erl

```diff
@@ -15,12 +15,18 @@
 %%% as verification of those proofs.
 -record(node, {
 	id,
-	type = branch,	% root | branch | leaf
-	data,			% The value (for leaves).
-	note,			% The offset, a number less than 2^256.
-	left,			% The (optional) ID of a node to the left.
-	right,			% The (optional) ID of a node to the right.
-	max,			% The maximum observed note at this point.
+	% root | branch | leaf
```

Comment:
I'd leave this the old way for the same reason as the newlines in long operators. To me the old version is easier to read and understand. It won't always be that way, but I think we should leave it up to the engineers maintaining

Reply:
Elvis is fine with this:
![2023-12-08 15 46 51](https://github.com/ArweaveTeam/arweave/assets/6074754/50e2478b-167c-45c8-8f56-7c844f2f812b)

Reply:
it's like the screenshot now